src/main/java/org/folio/edgecommonspring/security/SecurityManagerService.java

```diff
@@ -72,40 +71,42 @@ public ConnectionSystemParameters getParamsWithToken(String edgeApiKey) {
     return getParamsDependingOnCachePresent(salt, tenantId, username);
   }
 
-  private ConnectionSystemParameters getParamsDependingOnCachePresent(String salt, String tenantId,
+  public ConnectionSystemParameters getParamsDependingOnCachePresent(String salt, String tenantId,
```

Comment:
Does this need to be public?

Reply:
@steveellis  @Vignesh-kalyanasundaram found that similar method is being used in other edge-module, so made it public as per his suggestion and it can be directly utilized, I think!!

Reply:
Hi @steveellis , In Edge-Inn-Reach, the same logic is being executed when Authorization is passed in the header. So in order to use the library, we asked nirmal to make the method public. So that , we can call this method directly from Edge-Inn-Reach